src/app/zap-templates/zcl/data-model/chip/thread-network-diagnostics-cluster.xml

```diff
@@ -159,6 +164,15 @@ limitations under the License.
     <attribute side="server" code="0x3E" define="ACTIVE_THREAD_NETWORK_FAULTS" type="ARRAY" entryType="NetworkFault" length="4" writable="false" optional="false">ActiveNetworkFaultsList</attribute>
     <command source="client" code="0x00" name="ResetCounts" optional="true" cli="chip thread_network_diagnostics resetcounts">
       <description>Reception of this command SHALL reset the OverrunCount attributes to 0</description>
-    </command>           
+    </command>
+    <event side="server" code="0x00" name="ConnectionStatusEvent" priority="info" optional="true">
+      <description>Indicate that a Node’s connection status to a Thread network has changed</description>
+      <field id="0" name="ConnectionStatus" type="ConnectionStatus"/>
+    </event>
+    <event side="server" code="0x01" name="NetworkFaultChangeEvent" priority="info" optional="true">
+      <description>Indicate a change in the set of network faults currently detected by the Node Event</description>
+      <field id="0" name="Current" type="NetworkFault" array="true"/>
+      <field id="1" name="Previous" type="NetworkFault" array="true"/>
```

Comment:
This does not seem to match the spec.  The spec seems to be defining this as an event with a struct payload and that struct has fields?  But again, it's really hard to say because the formatting is so completely off from how events are supposed to be defined.  Spec issue filed?

Reply:
CHIP-Specifications/connectedhomeip-spec#4643